roxie/roxiemem/roxiemem.cpp

```diff
@@ -6490,6 +6492,16 @@ bool DataBuffer::attachToRowMgr(IRowManager *rowMgr)
     }
 }
 
+#ifdef TRACK_DATABUFF_STATE
+static constexpr const char * stateNames[(unsigned)DBState::max] = { "zero", "unowned", "queued", "attached", "freed" };
+void DataBuffer::changeState(DBState oldState, DBState newState)
```

Comment:
Can we add __FILE__,  __LINE__ and perhaps __func__ to these calls, some other tag to know which one the potential issue is from ?